### PR TITLE
missing icons

### PR DESCRIPTION
**Pull request by gela-d**
After https://github.com/bem/bem-components/commit/0b09f205e38cfd535c692ae1a0cfb36a713d90d0

### CODE REVIEW

File: common.blocks/icon/icon.styl [hunk 1]
```diff
@@ -1,5 +1,7 @@
 .icon
 {
+    position: relative;
```
Comment (mishanga) [inline common.blocks/icon/icon.styl]: Это нужно только внутри .button_theme_normal.

**Comment by gela-d** [comment]
@mishanga done

**Comment by mishanga** [comment]
:+1:

**Pull request merged**